### PR TITLE
Modal and rating

### PR DESCRIPTION
**Pull request by rdthetha**
Added rating and modal components

### CODE REVIEW

File: components/modal/modal.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
+#modal-container{
+
```
Comment (Ankur9669) [inline components/modal/modal.css]: Try to not use # selector

File: components/modal/modal.html [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+            class="code-iframe"
+            sandbox="allow-scripts allow-same-origin">
+        </iframe>
+        </section>
```
Comment (Ankur9669) [inline components/modal/modal.html:76]: Please indent the code in iframe

File: components/rating/rating.css [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  }
+  .star-wrapper:not(:checked) > label:hover,
+  .star-wrapper:not(:checked) > label:hover ~ label{
+    color: var(--primary-gold);;
+  }
+  .star-wrapper input:checked > label:hover,
+  .star-wrapper input:checked > label:hover ~ label{
+    color: var(--primary-gold);;
+    text-shadow: 1px 1px goldenrod;
+  }
```
Comment (Ankur9669) [inline components/rating/rating.css:36]: Does text-shadow works??

File: components/rating/rating.css [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+  .star-wrapper #three-star:checked ~ .star-result:before{
+    content: "Average";
+  }
+  .star-wrapper #two-star:checked ~ .star-result:before{
+    content: "Poor";
+  }
+  .star-wrapper #one-star:checked ~ .star-result:before{
+    content: "Very Poor";
+  }
```
Comment (Ankur9669) [inline components/rating/rating.css:68]: You have used # here it has very high specificity

**Review by Ankur9669 — CHANGES_REQUESTED** [review]
image
1. The Modal is not centered for the whole page, instead it is centered for only the content-container.
2. Also the background of the modal should be gray color


Inputs:
 The Inputs looks great to me!!!

File: components/modal/modal.css (hunk 1 above)
Comment (rdthetha) [reply components/modal/modal.css]: only for ratings component I used it!

File: components/rating/rating.css (hunk 3 above)
Comment (rdthetha) [reply components/rating/rating.css:36]: yes it gives a little color to the stars

File: components/modal/modal.html (hunk 2 above)
Comment (rdthetha) [reply components/modal/modal.html:76]: ok would fix it!

File: components/rating/rating.css (hunk 4 above)
Comment (rdthetha) [reply components/rating/rating.css:68]: Yes would keep in mind that!

**Review by omkarpatke — APPROVED** [review]
do work on the suggestion of Ankur otherwise, all code looks good for me.

**Comment by prankurpandeyy** [comment]
the code looks good to me .

**Pull request merged**